packages/gitgraph-react/package.json

```diff
@@ -49,23 +49,26 @@
     "react-dom": "^16.2.0"
   },
   "devDependencies": {
-    "@storybook/addon-actions": "^3.3.15",
-    "@storybook/addon-knobs": "^3.3.15",
-    "@storybook/addons": "^3.3.15",
-    "@storybook/react": "^3.3.15",
+    "@babel/core": "^7.1.0",
+    "@emotion/core": "^10.0.7",
+    "@emotion/styled": "^10.0.7",
```

Comment:
`@emotion/{core,styled}` seems to be necessary for the moment with Storybook 5. Otherwise, it throws an error on compilation.